cmd/cost/get.go

```diff
@@ -0,0 +1,244 @@
+package cost
+
+import (
+	"fmt"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+	"log"
+	"strconv"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/costexplorer"
+	"github.com/aws/aws-sdk-go/service/organizations"
+
+	"github.com/spf13/cobra"
+
+	awsprovider "github.com/openshift/osd-utils-cli/pkg/provider/aws"
+)
+
+// getCmd represents the get command
+func newCmdGet(streams genericclioptions.IOStreams) *cobra.Command {
+	ops := newGetOptions(streams)
+	getCmd := &cobra.Command{
+		Use:   "get",
+		Short: "Get total cost of a given OU. If no OU given, then gets total cost of v4 OU.",
+		Run: func(cmd *cobra.Command, args []string) {
+
+			cmdutil.CheckErr(opsCost.complete(cmd, args))
+			org, ce, err := opsCost.initAWSClients()
+			cmdutil.CheckErr(err)
+
+			//Get Organizational Unit
+			OU := organizations.OrganizationalUnit{Id: aws.String(ops.ou)}
+			//Store cost
+			var cost float64 = 0
```

Comment:
I wanted to get a precise representation of spendings, so I used float64. If I used an int, since we're getting and returning the cost of each account under a given OU (could be up to thousands of accounts), I thought we might be rounding too much and losing precision.

Reply:
I think you can just use `var cost float64` and it will initialize to 0